ui/components/multichain/select-action-modal-item/select-action-modal-item.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  AlignItems,
+  BackgroundColor,
+  Display,
+  FlexDirection,
+  IconColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import {
+  AvatarIcon,
+  AvatarIconSize,
+  Box,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../component-library';
+import Tooltip from '../../ui/tooltip';
+
+export const SelectActionModalItem = ({
+  actionIcon,
+  onClick,
+  showIcon,
+  primaryText,
+  secondaryText,
+  isDisabled,
+  tooltipTitle,
+}) => {
+  const SelectActionItem = () => {
+    return (
+      <Box
+        paddingTop={4}
+        paddingBottom={4}
+        gap={4}
+        display={Display.Flex}
+        flexDirection={FlexDirection.Row}
+        as="a"
+        href="#"
+        onClick={(e) => {
+          e.preventDefault();
+          onClick();
+        }}
+        className={`select-action-modal-item ${
```

Comment:
This can be an object:

```
className={classnames('select-action-modal-item', { 'select-action-modal-item__disabled': isDisabled })}
```

Reply:
We are not using disabled state now, instead removing this component for the network when it's not available. Updated here https://github.com/MetaMask/metamask-extension/pull/20559/commits/8d602620ce1bdf9f6b57d9e53f40aa8f8cb9580e